server/routes/approvingLicences/handlers/approvalCases.ts

```diff
@@ -38,6 +38,7 @@ export default class ApprovalCaseRoutes {
           name: convertToTitleCase(`${c.nomisRecord.firstName} ${c.nomisRecord.lastName}`.trim()),
           prisonerNumber: c.nomisRecord.prisonerNumber,
           probationPractitioner: c.probationPractitioner,
+          submittedBy: '',
```

Comment:
Will populate submittedBy field once backend work is done.

Reply:
I'm not sure I'm particularly a fan of merging a default blank value. What backend work needs to be done? If it's achievable in a reasonable timeframe, I'd argue it should be completed first.

Reply:
Gurps is working on this now.
It might be done by end of this week as he is on leave till Wednesday.
I think it's better to release as this functionality can't be used until Hardstop is live.
And I will integrate immediately once [backend ticket](https://dsdmoj.atlassian.net/browse/CVSL-1741) is done.